internal/export/client/init_test.go

```diff
@@ -0,0 +1,23 @@
+//
```

Comment:
This test does not make any sense now that memory db is moved out of this package.